build.msbuild

```diff
@@ -1,4 +1,4 @@
-<Project>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@jbagga If I do not add this, I get the following error when running `build.cmd`

```
"C:\github\jquery-validation-unobtrusive\build.msbuild" (default target) (1) ->
  C:\github\jquery-validation-unobtrusive\build.msbuild(1,1): error MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If t
he project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the pr
oject has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.
```

Reply:
I am unable to repro locally. That said, including it is fine!